### ISSUE TITLE
Short-Range Wireless ME Cables

### ISSUE THREAD

**Issue by TheExperiment1** [AppliedEnergistics/Applied-Energistics-2#4642]
Wireless ME cables would connect to a nearby wireless access point to allow for less cluttered networks at the cost of increased power drain. This would be less expensive and bulky than the quantum network bridge, but have less range as well.

This is the logical progression from the wireless terminal. A new block or custom treated cable would fit nicely with the rest of the mod.

**Comment by Pingger** [comment]
I like the idea... But there are some issues, that need to be discussed:
1. What is that range limit? (4, 6, 8, 10 Blocks)
1. Can it be upgraded using the range cards?
1. How many channels does it have? (It would propose 4 Channels, to make players prefer wired ME)
1. Would it transmit power? (I would propose: Only after connected, so it would need an energy acceptor to start both side
1. What about P2P tunnels? (yes/no/limit to something less than the channel limit/make it either 4 channels or P2P?)
1. Does the energy cost go up linearly or quadratic? (I would propose quadratic as it is (roughly) the same as in the real world)

**Comment by yueh** [comment]
Less bulky and less expensive is usually the reason something is rejected, if there is something similar already in AE2 like QNBs.

This is also mostly a duplicate of #192. 
In summary if we ever add it, it will be something like:

* Part which can be attached to cables
* Limited range (maybe 1-2 chunks?)
* Only in cardinal directions
* Requires line of sight
